perses/annihilation/relative.py

```diff
@@ -4140,6 +4140,7 @@ def _copy_nonbondeds(self):
             - CustomNonbondedForce for electrostatics interactions (not exceptions)
             - CustomNonbondedForce for sterics interactions (not exceptions)
             - NonbondedForce for reciprocal space electrostatic interactions and exceptions
+            - NonbondedForce for non-scaled steric interactions and exceptions
```

Comment:
Are the names supposed to be unique? Or is this the same thing:
`NonbondedForce for reciprocal space electrostatic interactions, non-scaled steric interactions and exceptions`

Reply:
The names are unique, I was just listing the type of force here, but I can add the names too.